### PR TITLE
Initial intro screen

### PR DESCRIPTION
**Pull request by uhalder**
This is just the UI for the IntroScreen.

Screen Shot 2023-07-19 at 11 18 33 PM

### CODE REVIEW

File: app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/introduction/IntroScreen.kt [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        )
+        IPDescriptionRow(
+            title = stringResource(id = R.string.resource_title),
+            body = stringResource(id = R.string.resource_description),
+            icon = R.drawable.ic_resource_24,
+            contentDescription = "resource_icon"
+        )
+        Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.dimension_44dp)))
+        IPFilledButton(
+            text = "Continue",
```
Comment (Audienix) [inline app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/introduction/IntroScreen.kt]: keep this in strings.xml?

File: app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/introduction/IntroScreen.kt [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+@Composable
+fun IntroScreen() {
+    Column(
+        modifier = Modifier
+            .padding(dimensionResource(id = R.dimen.dimension_32dp))
+            .fillMaxSize(),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.dimension_16dp))
+    ) {
+        Text(
```
Comment (Audienix) [inline app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/introduction/IntroScreen.kt]: Use IPHeader for this ?

File: app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/introduction/IntroScreen.kt [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            modifier = Modifier.fillMaxWidth(),
+            text = stringResource(id = R.string.intro_screen_heading1),
+            style = typography.headlineLarge,
+            lineHeight = 45.sp,
+            textAlign = TextAlign.Center
+        )
+        Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.dimension_32dp)))
+        Text(stringResource(id = R.string.intro_screen_heading2), fontSize = 24.sp)
```
Comment (Audienix) [inline app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/introduction/IntroScreen.kt]: Use IPHeader for this ?

File: app/src/main/kotlin/com/twain/interprep/presentation/ui/components/generic/IPDescriptionRow.kt [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    IPDescriptionRow(title = stringResource(id = R.string.note_title), body = stringResource(id = R.string.note_description), icon = R.drawable.ic_note_24, contentDescription = "note_icon")
+}
+
+@Preview
+@Composable
+fun IPIntroductionRow3Preview() {
+    IPDescriptionRow(title = stringResource(id = R.string.resource_title), body = stringResource(id = R.string.resource_description), icon = R.drawable.ic_resource_24, contentDescription = "resource_icon")
+}
+
+
```
Comment (Audienix) [inline app/src/main/kotlin/com/twain/interprep/presentation/ui/components/generic/IPDescriptionRow.kt]: remove extra line.

File: app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/introduction/IntroScreen.kt (hunk 1 above)
Comment (uhalder) [reply app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/introduction/IntroScreen.kt]: fixed this.

File: app/src/main/kotlin/com/twain/interprep/presentation/ui/components/generic/IPDescriptionRow.kt (hunk 4 above)
Comment (uhalder) [reply app/src/main/kotlin/com/twain/interprep/presentation/ui/components/generic/IPDescriptionRow.kt]: fixed this.

File: app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/introduction/IntroScreen.kt (hunk 2 above)
Comment (uhalder) [reply app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/introduction/IntroScreen.kt]: fixed.

File: app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/introduction/IntroScreen.kt (hunk 3 above)
Comment (uhalder) [reply app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/introduction/IntroScreen.kt]: fixed

**Review by Audienix — APPROVED** [review]

**Pull request merged**